keyboards/splitkb/kyria/keymaps/travishi/keymap.c

```diff
@@ -0,0 +1,575 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Sure, I'll add a copy of the GNU license the keyboard creator us using